### PR TITLE
Restaurant Website

### PR DESCRIPTION
**Pull request by tiagoit**
# Project: Microverse | The Odin Project | JS | 03 | Mexican Restaurant

- Project setup with Webpack, Eslint, Prettier, Stickler-ci, bootstrap.
- Partial modules for header, main, footer, menu-tab and contact-tab.
- Method to set active tab.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="Nice Restaurant Page">
+  <meta name="author" content="Tiago Brito">
+  <title><%= htmlWebpackPlugin.options.title %></title>
+</head>
+<body></body>
```
Comment (bellom) [inline src/index.html]: Here in; according to the instruction we are told to leave `<div id="content">` tags right ..... and append each new element to the `div#content` once the page is first loaded.
image
https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ferreirati  :clap: 
### Awesome job so far!! :+1: 
#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1:, just few issues left to work on............ :slightly_smiling_face: 
Kindly work on the following, i decided not to mark this PR as invalid, seems you're on the right track, you just need to add more information and also check the comment below;

**1. README file not descriptive enough.**
image
**_Example: (steps needed to `git clone` your repo should be included for the newbie and also user should be able to run the live version of your App :smile: )_**
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/

**2. Pull Request has no meaningful title and a short description;**
image
Example; 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**3. When a tab is clicked the content is replaced.**
As soon as i clicked the buttons from your webpage below, the content wasn't replaced.
image

image
https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

**_Kindly go through both links up there for you to understand the difference between `README` and `Pull Request title and short summary`_**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by tiagoit** [comment]
> ## Hi @ferreirati clap
> ### Awesome job so far!! +1
> #### You have done wonderfully well, you passed `almost` all the checklist for this project +1, just few issues left to work on............ slightly_smiling_face
> 
> Kindly work on the following, i decided not to mark this PR as invalid, seems you're on the right track, you just need to add more information and also check the comment below;
> 
> **1. README file not descriptive enough.**
> image
> **_Example: (steps needed to `git clone` your repo should be included for the newbie and also user should be able to run the live version of your App smile )_**
> image
> https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/
> 
> **2. Pull Request has no meaningful title and a short description;**
> image
> Example;
> image
> https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/
> 
> **3. When a tab is clicked the content is replaced.**
> As soon as i clicked the buttons from your webpage below, the content wasn't replaced.
> image
> 
> image
> https://www.theodinproject.com/courses/javascript/lessons/restaurant-page
> 
> **_Kindly go through both links up there for you to understand the difference between `README` and `Pull Request title and short summary`_**
> 
> **Kindly re-submit for review after making all those changes slightly_smiling_face Thumps -up await you.....**
> 
> Best,
> 
> **Oluwaseun Bello**
> **_Slack Handle - @sean bellom_**

Hello Bellom, can u try starting it with npm start, I douBle checked here also with my coding partner, the taBs r Being replaced.

**Comment by bellom** [comment]
Yes, that was exactly what i did, we can jump on call then....
already messaged you on slack, kindly get back to me ASAP..... and kindly work on the comment i left on `src/index.html` line 10.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ferreirati  :clap: 

**Great work making some of the necessary corrections from the previous review, just two issues i mentioned earlier left**

**Kindly watch and listening carefully to the video and audio code review below;**

https://www.loom.com/share/7b52dd6534e3423fa8e7b71e2fd6b751

**_Hope it helps clarify the issues we are having ......_**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by tiagoit** [comment]
> ## Hi @ferreirati clap
> 
> **Great work making some of the necessary corrections from the previous review, just two issues i mentioned earlier left**
> 
> **Kindly watch and listening carefully to the video and audio code review below;**
> 
> https://www.loom.com/share/7b52dd6534e3423fa8e7b71e2fd6b751
> 
> **_Hope it helps clarify the issues we are having ......_**
> 
> Best,
> 
> **Oluwaseun Bello**
> **_Slack Handle - @sean bellom_**

Oh, I see, the intended behavior is to simulate SPA routes, the actual behavior is the way I intended to code this, but I understood now how it should be. I'm implementing the fixes now.

PS: Amazing PR Bello, tks a lot!

**Review by bellom — APPROVED** [review]
## Hi @ferreirati  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**